tests/unit/test.config.js

```diff
@@ -474,6 +505,40 @@ describe('config', () => {
       assert.strictEqual(newArgv.apiKey, cmdApiKey);
     });
 
+    it('can load multiple configs', () => {
```

Comment:
This test looks pretty similar to the one at line 252, but on the sign command instead of a fake command, aand with commantOpt instead of globalOpt. 

if we need both these tests, it would be nice to extract the common part into a test helper function defined in this file.

Reply:
Yeah, they are both necessary. One will apply a global option twice and the other will apply a sub-command option twice. The sub-command test covers some recursion.

I added some comments and improved the test specs to make it more clear. I couldn't really see any code that could be abstracted in a helpful way.

Reply:
:+1: the comments and the changes to the test names are useful and more then enough to quickly notice how the two tests are different in purpose (so that we don't end up to remove them by mistake by missing their different purpose), looks good to me.